lib/cadet_web/controllers/auth_controller.ex

```diff
@@ -56,33 +52,60 @@ defmodule CadetWeb.AuthController do
     send_resp(conn, :bad_request, "Missing parameter")
   end
 
+  @doc """
+  Receives a /refresh request with valid attribute.
+
+  Exchanges the refresh_token with a new access_token.
+  """
   def refresh(conn, %{"refresh_token" => refresh_token}) do
-    case Guardian.exchange(refresh_token, "refresh", "access") do
-      {:ok, {refresh_token, _}, {access_token, _}} ->
+    # TODO: Refactor to use refresh after guardian_db > v1.1.0 is released.
+    case Guardian.resource_from_token(refresh_token) do
+      {:ok, user, %{"typ" => "refresh"}} ->
+        %{access_token: access_token, refresh_token: refresh_token} = generate_tokens(user)
         render(conn, "token.json", access_token: access_token, refresh_token: refresh_token)
```

Comment:
What do you think about just doing:

`render(conn, "token.json", generate_tokens(user))`
or
```
tokens = generate_tokens(user)
render(conn, "token.json", tokens)
```

Reply:
Oops. You're right. Will fix this